src/disco/css/App.scss

```diff
@@ -21,7 +22,57 @@ img {
 }
 
 .disco-pane {
-  padding: 50px;
+  box-sizing: content-box;
+  padding: 50px 20px;
   margin: 0 auto;
   max-width: 680px;
 }
+
+header {
+  display: flex;
+  border-bottom: 2px solid #ddd;
+  flex-direction: row;
+  margin-bottom: 40px;
+  padding-bottom: 40px;
+
+  .content {
+    flex-grow: 1;
+  }
+
+  h1 {
+    display: block;
+    letter-spacing: 0.11ex;
+    line-height: 1.5;
+    font-size: 22px;
+    font-weight: 400;
+    margin: 0 0 20px;
+  }
+
+  p:last-child {
+    display: block;
+    margin-bottom: 0;
+  }
+
+  .play-video {
+    background: #ccc;
+    border-radius: 5px;
+    display: block;
+    flex-grow: 0;
+    margin-left: 50px;
```

Comment:
This is going to need an alternate rule for RTL. If you express this in flex it will work though. The spec defines a bunch of specific values for these sizes so I think it would be nice to set those in variables and then let flex lay it out correctly.

```scss
$blockWidth: 680px;

header {
  $videoWidth: 215px;
  $videoHeight: 120px;
  $spaceBetween: 50px;

  display: flex;
  justify-content: space-between;
  width: $blockWidth;
  
  .content {
    width: $blockWidth - $videoWidth - $spaceBetween;
  }

  .play-video {
    display: block;
    height: $videoHeight;
    width: $videoWidth;
  }
}